vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -1182,6 +1187,18 @@ def fused_experts(hidden_states: torch.Tensor,
             a2_scale=a2_scale,
             apply_router_weight_on_input=apply_router_weight_on_input,
         )
+    elif (allow_cutlass_block_scaled_grouped_gemm and use_fp8_w8a8
+            and _valid_cutlass_block_scaled_grouped_gemm(hidden_states, w1, w2)):
+        assert apply_router_weight_on_input is False
+        return run_cutlass_block_scaled_fused_experts(
+            a=hidden_states,
+            w1=w1,
+            w2=w2,
+            w1_scale=w1_scale,
+            w2_scale=w2_scale,
+            topk_weights=topk_weights,
+            topk_ids=topk_ids
+        )
```

Comment:
Would it make sense for the usage of CUTLASS to be decided on the quantized method level? For example, the same way CUTLASS MoE is picked in `CompressedTensorsMoEMethod`'s `get_moe_method()` function